xamarin/Xamarin.Forms
# [Handlers] Build controlgallery and UItests for it

### Description of Change ###

This goes on top of #12546 . As we build Handlers we should make sure the old UItests for that work, we also should migrate the UITests for that control to the new xUNIT tests



### Issues Resolved ### 

- None

### API Changes ###
 
 None

### Platforms Affected ### 

- Core/XAML (all platforms)
- iOS
- Android
- UWP

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 


Not applicable

### Testing Procedure ###

See if UITests run on the new  apps on AppCenter , XamarinFormsHandlers.Android and XamarinFormsHandlers.iOS

### PR Checklist ###
